README.md

```diff
@@ -0,0 +1 @@
+A react native application with todo implementation.
```

Comment:
We generally add `TODO` now you have implemented. 😅

Reply:
😅